spill/main.c

```diff
@@ -1,15 +1,46 @@
 #include "snake.h"
 #include "board.h"
+#include "constants.h"
 #include <stdio.h>
 
 void main() {
+    int board[BOARD_HEIGHT][BOARD_WIDTH];
+    int count_food = 0;
+    int snake_id_counter = 0;
     init_board();
```

Comment:
Doesnt this board override the global board in this scope? how does init_board interact with this?